ntoskrnl/mm/balance.c

```diff
@@ -284,6 +296,7 @@ MmRequestPageMemoryConsumer(ULONG Consumer, BOOLEAN CanWait,
 
     /* Update the target */
     InterlockedIncrementUL(&MiMemoryConsumers[Consumer].PagesUsed);
+    UpdatePeakCommitment();
```

Comment:
Actually, I think we're missing a matching update in `MmReleasePageMemoryConsumer` where this gets decremented again.

Reply:
yes